data/tlg2001/tlg043/__cts__.xml

```diff
@@ -1,7 +1,7 @@
-<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg2001" xml:lang="grc" urn="urn:cts:greekLit:tlg2001.tlg043">
-  <ti:title xml:lang="eng">In Libros Aristotelis De Caelo Paraphrasis</ti:title>
-  <ti:edition urn="urn:cts:greekLit:tlg2001.tlg043.opp-grc1" workUrn="urn:cts:greekLit:tlg2001.tlg043">
-    <ti:label xml:lang="eng">In Libros Aristotelis De Caelo Paraphrasis</ti:label>
-    <ti:description xml:lang="eng">Themistius, In Libros Aristotelis De Caelo Paraphrasis</ti:description>
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg2001" xml:lang="lat" urn="urn:cts:greekLit:tlg2001.tlg043">
+  <ti:title xml:lang="lat">In Libros Aristotelis De Caelo Paraphrasis</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg2001.tlg043.opp-lat1" workUrn="urn:cts:greekLit:tlg2001.tlg043">
```

Comment:
Since this text is not in the original language of the work, it should be marked as `<ti:translation>` and an `xml:lang="lat"` attribute should be added to it.